eras/conway/impl/cardano-ledger-conway.cabal

```
@@ -268,7 +343,7 @@ executable gen-golden
     cardano-ledger-conway,
     testlib,
 
-test-suite tests
+test-suite testtestss
```

Accidental stuttering
```suggestion
test-suite tests
```